exchange/targeting_test.go

```diff
@@ -15,28 +15,30 @@ import (
 	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
+// Using this set of bids in more than one test
```

Comment:
You could probably undo all the changes to `exchange/targeting_test.go` if you wanted, but... *shrugs*. Not the biggest deal. Just more complicated now than it needs to be.